src/components/adaptive-cards/ActionOpenURL/ActionOpenUrl.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {acPropTypes, registerComponent} from '../Component/Component';
+import webexComponentClasses from '../../helpers';
+import {Button} from '../../generic';
+
+/**
+ * Adaptive Cards Action.OpenUrl component
+ * https://adaptivecards.io/explorer/Action.OpenUrl.html
+ *
+ * @param {object} props  React props passed to the component
+ * @param {object} props.data  Active cards definition
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {object} [props.style]  Custom style to apply
+ * @returns {object} JSX of the component
+ */
+export default function ActionOpenUrl({data, className, style}) {
+  const [cssClasses, sc] = webexComponentClasses('ac-action', className, {
+    'open-url': true,
+    disabled: data.isEnabled === false,
+  });
+
+  let buttonType;
+
+  if (data.style === 'positive') {
+    buttonType = 'join-outline';
+  } else if (data.style === 'destructive') {
+    buttonType = 'cancel-outline';
+  } else {
+    buttonType = 'primary';
+  }
+
+  return (
+    <Button
+      className={cssClasses}
+      isDisabled={data.isEnabled === false}
+      onClick={() => window.open(data.url, '_blank')}
```

Comment:
can you try providing an url like "javascript:alert('injection')"? if it displays the prompt, we need to validate data.url as a real url, eg: https://stackoverflow.com/a/43467144/210550